go.mod

```diff
@@ -61,6 +61,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require gopkg.in/yaml.v3 v3.0.1
```

Comment:
This can go here https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/bf820a63a7751b75527b5c233c71c67f69457cd5/go.mod#L5